pkg/addons/addons.go

```diff
@@ -220,12 +223,20 @@ func writeDB(client *client, db map[string]unstructured.Unstructured) error {
 }
 
 func Main(cs *acsapi.ContainerService, c *config.Config, dryRun bool) error {
-	client, err := newClient(dryRun)
+	client, restconfig, err := newClient(dryRun)
 	if err != nil {
 		return err
 	}
 
-	if err := client.waitForHealthz(); err != nil {
+	transport, err := rest.TransportFor(restconfig)
+	if err != nil {
+		return err
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
```

Comment:
just use context.Background() here for now, no timeout

Reply:
done.